src/consul/consul_types.bal

```diff
@@ -97,5 +98,5 @@ public type Value record {
 public type ConsulConfiguration record {
     string uri = "";
     string aclToken = "";
```

Comment:
We can make these optional fields and remove all the `if (config.aclToken != "")` checks everywhere, maybe in a separate iteration.